src/cycax/cycad/cycad_part.py

```diff
@@ -555,3 +555,27 @@ def level(
         if subtract:
             for my_side, other_side in level_tasks:
                 self.assembly.subtract(other_side, self)
+
+    def rotate(self, actions: str):
+        """
+        This can be used to rotate a part in the assembly as follows:
+        CycadPart.rotate("xxyzyy")
+        This is: 2 rotate_freeze_front, rotate_freeze_left, rotate_freeze_top, 2 rotate_freeze_left.
+        Args:
+            actions: This is a string specifying rotations.
+
+        Raises:
+            ValueError: When the given actions contains a strign that is non x, y, or z.
+        """
+        for action in actions:
```

Comment:
I think it would be kind to add a `.lower()` so it is case insensitive.

Reply:
See PR #58